Sources/FluentKit/Model/Schema.swift

```diff
@@ -11,7 +11,7 @@ extension Schema {
         self.alias ?? self.schema
     }
     
-    static var spaceIfNotAliased: String? {
-        return self.alias == nil ? self.space : nil
+    public static var spaceIfNotAliased: String? {
```

Comment:
Is this intended?

Reply:
It certainly is. But I completely forget to mention it in the commit log or the PR description 😅